_layouts/project.html

```diff
@@ -23,36 +23,94 @@ <h2>{{ page.title }}</h2>
     <div class="well">
       <table class='table table-condensed'>
         <tbody>
-          {% if page.website %}
+            <tr>
+              <td>Project Status</td>
+              <td><span class='label label-primary'>{{page.project_status}}</span></td>
+            </tr>
+          {% if page.app_link %}
           <tr>
             <td>Website</td>
-            <td><a target='_blank' href='{{page.website}}'>{{page.title}}</a></td>
+            <td><a target='_blank' href='{{page.app_link}}'>{{page.app_link}}</a></td>
           </tr>
           {% endif %}
+          {% if page.repos %}
           <tr>
-            <td>Tags</td>
+            <td>Code</td>
+            <td>{% for repo in page.repos %}{% if forloop.first != true %} <br />{% endif %}<a target='_blank' href='{{repo}}'>{{repo}}</a>{% endfor %}</td>
```

Comment:
Nitpicky, but could we use a `<ul class="unstyled">` rather than `<br />`s?

Reply:
This'll also let you drop the `forloop.first` logic.

Reply:
Yeah, I'll change this, but I actually want to change the whole table altogether.